frontend/src/host_orchestrator/orchestrator/instancemanager.go

```diff
@@ -235,7 +235,7 @@ func defaultMainBuild() *apiv1.AndroidCIBuild {
 
 func (m *CVDToolInstanceManager) launchFromAndroidCI(
 	req apiv1.CreateCVDRequest, op apiv1.Operation) (*apiv1.CVD, error) {
-	var mainBuild, kernelBuild *apiv1.AndroidCIBuild = defaultMainBuild(), nil
+	var mainBuild, kernelBuild, bootloaderBuild *apiv1.AndroidCIBuild = defaultMainBuild(), nil, nil
```

Comment:
nit: please use 3 lines. With longer variable names the code is less readable and the relative savings in space and typing are minuscule.

Reply:
done